src/core/Timepicker/BaseTimePicker.tsx

```diff
@@ -0,0 +1,26 @@
+import 'date-fns';
+import React, { useState } from 'react';
+import MomentUtils from '@date-io/moment';
+import { MuiPickersUtilsProvider, TimePicker } from '@material-ui/pickers';
+import { BaseTimePickerProps } from './base';
+
+interface TimePickerProps extends BaseTimePickerProps {
+  value?: Date;
+}
+
+const BaseTimePicker: React.FC<TimePickerProps> = ({ value }) => {
+  const date = value != undefined ? value : new Date();
+  const [selectedDate, handleDateChange] = useState<Date | null>(date);
+  return (
+    <MuiPickersUtilsProvider utils={MomentUtils}>
+      <TimePicker
+        autoOk
+        okLabel="Save"
+        value={selectedDate}
+        data-testid="timepicker"
+        onChange={(newDate: any) => handleDateChange(newDate)}
+      />
+    </MuiPickersUtilsProvider>
```

Comment:
Can we use a simple KeyboardTimePicker instead! And avoid the MUI utility wrapper if possible!

```
<KeyboardTimePicker
          margin="normal"
          id="time-picker"
          label="Time picker"
          value={selectedTime}
          onChange={handleTimeChange}
          KeyboardButtonProps={{
            'aria-label': 'change time',
          }}
        />
```